bitnami/chainloop/values.yaml

```diff
@@ -0,0 +1,2254 @@
+
+  ## ServiceAccount configuration for the Controlplane Migration
+  ##
+  ## @param controlplane.serviceAccount.create Specifies whether a ServiceAccount should be created
+  ## @param controlplane.serviceAccount.annotations Annotations for service account. Evaluated as a template. Only used if `create` is `true`.
+  ## @param controlplane.serviceAccount.name Name of the service account to use. If not set and create is true, a name is generated using the fullname template.
+  ## @param controlplane.serviceAccount.automountServiceAccountToken Mount Service Account token in controlplane pods
+  serviceAccount:
+    create: true
+    annotations: {}
+    name: ""
+    automountServiceAccountToken: false
+
+  ## @section Control Plane Database
+
+  ## @extra controlplane.externalDatabase External PostgreSQL configuration. These values are only used when postgresql.enabled is set to false
+  ## @param controlplane.externalDatabase.host Database host
+  ## @param controlplane.externalDatabase.port Database port number
+  ## @param controlplane.externalDatabase.user Non-root username
+  ## @param controlplane.externalDatabase.database Database name
+  ## @param controlplane.externalDatabase.password Password for the non-root username
+  ##
+  externalDatabase:
+    host: ""
+    port: 5432
+    user: ""
+    database: ""
+    password: ""
+
+  ## @section Control Plane Authentication
+  auth:
+    ## @param controlplane.auth.passphrase Passphrase used to sign the Auth Tokens generated by the controlplane. Leave empty for auto-generation
+    ##
+    passphrase: ""
+
+    ## @param controlplane.auth.oidc.url Full authentication path, it should match the issuer URL of the Identity provider (IDp)
+    ## @param controlplane.auth.oidc.clientID OIDC IDp clientID
+    ## @param controlplane.auth.oidc.clientSecret OIDC IDp clientSecret
+    ## @param controlplane.auth.oidc.loginURLOverride Optional OIDC login URL override, useful to point to custom login pages
+    ## @param controlplane.auth.oidc.externalURL Optional External URL for the controlplane to the outside world
+    oidc:
+      url: ""
+      clientID: ""
+      clientSecret: ""
+      loginURLOverride: ""
+      externalURL: ""
+
+    ## @extra controlplane.auth.allowList.rules List of domains or emails to allow
+    ## @extra controlplane.auth.allowList.selectedRoutes List of selected routes to allow. If not set it applies to all routes
+    ## @extra controlplane.auth.allowList.customMessage Custom message to display when a user is not allowed
+    #    allowList:
+    #      rules: []
+    #      selectedRoutes: []
+    #      customMessage: ""
+
+  ## @section Control Plane Networking
+  service:
+    ## @param controlplane.service.type Service type
+    type: ClusterIP
+
+    ## @param controlplane.service.ports.http controlplane service HTTP port
+    ## @param controlplane.service.ports.https controlplane service HTTPS port
+    ##
+    ports:
+      http: 80
+      https: 443
```

Comment:
I think you missed this from the previous review.
Please remove the `https` values since they are not being used.